geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/index/GeoMesaSchemaValidator.scala

```diff
@@ -20,10 +20,31 @@ object GeoMesaSchemaValidator {
   def validate(sft: SimpleFeatureType): Unit = {
     MixedGeometryCheck.validateGeometryType(sft)
     TemporalIndexCheck.validateDtgField(sft)
+    ReservedWordCheck.validateAttributeNames(sft)
   }
 }
 
 /**
+  * Utility object that ensures that none of the (local portion of the) property
+  * names is a reserved word in ECQL.  Using those reserved words in a simple
+  * feature type will cause queries to fail.
+  */
+object ReservedWordCheck extends LazyLogging {
+
+  // ensure that no attribute names are reserved words within GeoTools that will cause query problems
+  def validateAttributeNames(sft: SimpleFeatureType): Unit = {
+    val reservedWords = FeatureUtils.sftReservedWords(sft)
+    if (reservedWords.nonEmpty) {
+      // TODO:  Make this an exception after GeoMesa 1.2.2
+      logger.warn(
+        "The simple feature type contains one or more attributes whose names are reserved words:  " +
```

Comment:
hehe...  And yes, it was formerly in the CMoS.  With the advent of WYSIWYG, it's dropped out of fashion, but code remains a fixed-width-font format, so I would *always* use two spaces after the colon.